storage/innobase/handler/ha_innodb.cc

```diff
@@ -15890,6 +15890,8 @@ int ha_innobase::discard_or_import_tablespace(bool discard,
     err = row_import_for_mysql(dict_table, table_def, m_prebuilt);
 
     if (err == DB_SUCCESS) {
+      dict_table_autoinc_set_col_pos(
```

Comment:
With normal table, after a restart, can you please how the autoin column position is set?

Reply:
Can you please get stack trace?

Reply:
@rahulmalik87 before the import, the table would have been opened and the autoin col pos is set. Do we destroy the opened table object?

Reply:
in another way, the question is, why it is unset at the time of import? Before import, table open would have set it

Reply:
As part of import we explicit set autoinc col pos to UNDEF inside row0import.cc 
`/* This should be set later in handler level, where we know the
  autoinc counter field index */
  table->autoinc_field_no = ULINT_UNDEFINED;`